src/web/client/WebExtensionContext.ts

```diff
@@ -52,74 +52,74 @@ class WebExtensionContext {
         currentSchemaVersion: ""
     };
 
-    public getPowerPlatformExtensionContext() {
-        return this.powerPlatformExtensionContext;
+    public getWebExtensionContext() {
```

Comment:
we can actually make this a getter function with name context. In any case we are accessing all of these as WebExtensionContext.<func>. Or, any better suggestions here?

Reply:
As discussed created a task https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3116420